infinitest-lib/pom.xml

```diff
@@ -55,5 +55,23 @@
 			<artifactId>commons-lang</artifactId>
 			<version>2.6</version>
 		</dependency>
+		<dependency>
+			<groupId>com.google.auto.value</groupId>
+			<artifactId>auto-value</artifactId>
+			<version>${autovalue.version}</version>
+			<scope>provided</scope>
```

Comment:
Auto value is an annotation processor and is only needed during compilation phase. So there is no need for it at runtime or in a p2 repository. (See [auto value user guid](https://github.com/google/auto/blob/master/value/userguide/index.md))